### ISSUE TITLE
Selecting text ought to pause scrolling (or further output in general), as happens in CMD window

### ISSUE THREAD

**Issue by dave-c-whitney** [microsoft/terminal#4399]
When I highlight text for selection, and ultimately for copy-to-clipboard, the window can keep scrolling with new output. The selection then isn't clear. Sometimes the window can scroll while I'm still in the act of selecting and then of course I'm no longer selecting what I wanted to select.

Once I click down and start text selection, no further output to the window should be allowed - not until the selection is cleared (either by pressing ESC, or right-click to copy).

**Comment by j4james** [comment]
This is probably a duplicate of #980.

**Comment by DHowett-MSFT** [comment]
It is! Thanks. /dup #980.
